sklearnex/linear_model/tests/test_linear.py

```diff
@@ -124,3 +128,15 @@ def test_sklearnex_reconstruct_model(dataframe, queue, dtype):
 
     tol = 1e-5 if _as_numpy(y_pred).dtype == np.float32 else 1e-7
     assert_allclose(gtr, _as_numpy(y_pred), rtol=tol)
+
+
+def test_sklearnex_multivariate_ridge_coefs():
+    from sklearnex.linear_model import Ridge
+
+    _test_multivariate_ridge_coefficients(Ridge, random_state=0)
+
+
+def test_sklearnex_multivariate_ridge_alpha_shape():

```

Comment:
Not the scope of this PR, but it make sense as a follow up extend it for the different dataframe inputs check.